conf/homeserver.yaml

```diff
@@ -1272,20 +1272,20 @@ enable_registration: __ALLOWED_ACCESS__
 # The user must provide all of the below types of 3PID when registering.
 #
 registrations_require_3pid:
- - email
+ - __REGISTRATIONS_REQUIRE_3PID__
 #  - msisdn
 
 # Explicitly disable asking for MSISDNs from the registration
 # flow (overrides registrations_require_3pid if MSISDNs are set as required)
 #
-#disable_msisdn_registration: true
+disable_msisdn_registration: __DISABLE_MSISDN_REGISTRATION__
 
 # Mandate that users are only allowed to associate certain formats of
 # 3PIDs with accounts on this server.
 #
-#allowed_local_3pids:
-#  - medium: email
-#    pattern: '^[^@]+@matrix\.org$'
+allowed_local_3pids:
+  - medium: __REGISTRATIONS_REQUIRE_3PID__
+    pattern: __ALLOWED_LOCAL_3PIDS__
 #  - medium: email
 #    pattern: '^[^@]+@vector\.im$'
 #  - medium: msisdn
```

Comment:
Here too how do you manage if I only want to mange both `email` and `msisdn` ?

Reply:
here, @thardev could contribute a `getter/setter` with some `sed`

Reply:
solved thanks to @rosbeef